randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1614,25 +1650,33 @@ Extra - map_name: rm_a5b16
   * Layers: default
   * Open Passage to Spiky Space Jump Trial Reward Room/Dock to Spiky Space Jump Trial
   > Dock to Spiky Space Jump Trial Entrance
-      All of the following:
-          # # TODO: hypermode walljump + expert damage boost + 700 damage suitless
-          Gravity Suit
-          Any of the following:
-              Space Jump
-              Damage Boost (Advanced) and Walljump (Advanced) and Normal Damage ≥ 150
+      Any of the following:
+          All of the following:
+              # With Gravity Suit
+              Gravity Suit
+              Any of the following:
+                  Space Jump
+                  Damage Boost (Advanced) and Walljump (Advanced) and Normal Damage ≥ 120
```

Comment:
a) howd you get on the 120 value
b) what about with HJ? that'd one probably be easier and take less health